packages/backend/src/graphql/resolvers/tenant.ts

```diff
@@ -174,6 +175,7 @@ export function tenantToGraphQl(tenant: Tenant): SchemaTenant {
     createdAt: new Date(+tenant.createdAt).toISOString(),
     deletedAt: tenant.deletedAt
       ? new Date(+tenant.deletedAt).toISOString()
-      : null
+      : null,
+    isOperator: tenant.apiSecret === Config.adminApiSecret
```

Comment:
Not really in the scope of this PR but related: https://github.com/interledger/rafiki/issues/3276